db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -61,7 +61,10 @@ class NonBatchedOpsStressTest : public StressTest {
           static_cast<int>(VerificationMethod::kNumberOfMethods);
 
       const VerificationMethod method =
-          static_cast<VerificationMethod>(thread->rand.Uniform(num_methods));
+          static_cast<VerificationMethod>(thread->rand.Uniform(
+              (FLAGS_use_put_entity_one_in > 0 || FLAGS_user_timestamp_size > 0)
```

Comment:
This might be a merge conflict; the condition related to `FLAGS_use_put_entity_one_in` got removed recently

Reply:
ah ok, let me update. Thanks for catching this!